apps/src/applab/constants.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
   FOOTER_HEIGHT: 30,
   DESIGN_ELEMENT_ID_PREFIX: 'design_',
-  NEW_SCREEN: "New screen..."
+  NEW_SCREEN: "New screen...",
+  MODE: {
+    CODE: 'CODE',
+    DESIGN: 'DESIGN'
```

Comment:
Are these strings user visible? If so, I'd prefer they live in our i18n pathway. If this is just an enum, this is fine.